### PR TITLE
[MesheryPlayComponent] - Refactorized class based meshery play component to functional component

### LINKED ISSUE #12169: [UI] Migrate Meshery Play Component to Functional Component with MUI v5 Styling and Sistent Component use
### Current Behavior
This [MesheryPlayComponent](https://github.com/meshery/meshery/blob/764d510d136d0e6640d7d5dc2f2e55c759c0be7c/ui/components/MesheryPlayComponent.js#L86) Component is a class-based component and it uses MUI v4's styling. These need to be updated for improved performance and consistency.

### Desired Behavior
- [ ]  migrate class-based component to a functional component
- [ ]  use [Sistent](https://github.com/layer5io/sistent) components for  MUI v4 to MUI v5 migration

Sistent is our design system that uses MUI v5, you can also refer to this [doc](https://docs.meshery.io/project/contributing/contributing-ui-sistent) to understand how to use Sistent.

### Acceptance Tests
The component must be updated without introducing any breaking changes in functionality or theming across all instances where it is used.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Pratik-050**
**Notes for Reviewers**

This PR fixes #12169

Migrated from class based component to functional component without breaking the normal functionality of the component.

- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by jerensl — APPROVED** [review]
Looks good to me, @SAHU-01 Wdyt?

File: ui/components/MesheryPlayComponent.js [hunk 1]
```diff
@@ -83,211 +83,184 @@ const styles = (theme) => ({
...
+  const handleRouteChange = useCallback(() => {
+    const queryParam = router.query.adapter;
     if (queryParam) {
-      const selectedAdapter = this.props.meshAdapters.find(
+      const selectedAdapter = meshAdapters.find(
         ({ adapter_location }) => adapter_location === queryParam,
       );
       if (selectedAdapter) {
-        this.setState({ adapter: selectedAdapter });
+        setAdapterState(selectedAdapter);
```
Comment (jerensl) [inline ui/components/MesheryPlayComponent.js:98]: Are u sure this state got updated?

https://react.dev/learn/updating-objects-in-state

**Review by jerensl — CHANGES_REQUESTED** [review]

File: ui/components/MesheryPlayComponent.js (hunk 1 above)
Comment (Pratik-050) [reply ui/components/MesheryPlayComponent.js:98]: Sorry My bad. Let me make some changes.

**Comment by l5io** [comment]
🚀 Preview for commit 5567f9988453d1b65cbb936e20cc27578a554e6b at: https://6700199f05f377669ff5ce92--meshery-docs-preview.netlify.app

File: ui/components/MesheryPlayComponent.js (hunk 1 above)
Comment (Pratik-050) [reply ui/components/MesheryPlayComponent.js]: Done the changes. Somehow the commit made by l5io is also included in the commit that I made.

**Comment by SAHU-01** [comment]
@Pratik-050 could you trace back to the commit history and remove the unwanted commits?Else you can open a new PR with your changes and I'd review the same.

We don't want the documentation commits to reflect in here.

**Comment by Pratik-050** [comment]
> @Pratik-050 could you trace back to the commit history and remove the unwanted commits?Else you can open a new PR with your changes and I'd review the same.
> 
> We don't want the documentation commits to reflect in here.

OK then I'll make a new PR as this one is so much cluttered.

**Pull request closed without merging**